### DISCUSSION TITLE
DB25 port on PCB

### DISCUSSION THREAD

**Discussion by AndyDiags** [erichelgeson/BlueSCSI-v1#53] [Ideas]
Is it possible to have a board designed where we can solder a DB25 connector directly onto the board?
I would like to solder a straight DB25 connector, so that I can plug in the BlueSCSI into a Macintosh Plus at 90 degrees.

I noticed there is a planned HW version 1.1-a, where "Other SCSI connection types" will be supported.  I hope that the DB25 is one of the other connection types.

Below are a couple samples of connectors that could be used, would most likely pick one which would allow for minimal clearance so that the board is as close to the Mac as possible.

https://www.digikey.sg/en/products/detail/te-connectivity-amp-connectors/2-5747704-0/1123863
https://www.digikey.sg/en/products/detail/amphenol-icc-commercial-products/L717SDB25POL2/4886643
https://www.digikey.sg/en/products/detail/te-connectivity-amp-connectors/2301835-1/7776534
https://www.digikey.sg/en/products/detail/te-connectivity-amp-connectors/5745968-2/1123844
https://www.digikey.sg/en/products/detail/te-connectivity-aerospace-defense-and-marine/1757827-3/2339464

<img width="615" alt="Screen Shot 2021-11-05 at 10 30 48 PM" src="https://user-images.githubusercontent.com/85074233/140518073-81db3994-f45a-4e39-a6d5-ea9dc408ccda.png">

Here's an example just to clarify the direct in which I would like to plug it in.  The Arduino could be facing inwards as well, either way would be fine.
![400439f087ddee6557a6965fbb72f944](https://user-images.githubusercontent.com/85074233/140520262-8813a9f6-36b2-4028-9c1b-03293e4dbfc6.jpg)

**Comment by erichelgeson** [comment]
Yes, 1.1-a will have a db-25 connection (in the mail from jlcpcb now) - but I had some trouble facing the bulepill in wards, there is just not enough clearance - I was able to get it close enough if i trimmed the jumpers off the bluepill! Also the Classic 1/2 have different celarances for that lip on the IO ports and it wont quite fit. 

https://www.tindie.com/products/jcs/right-angle-db25-adapter/ - is another neat option to make it right angle.

I expect to upload the db-25 1.1-a as soon as I verify they work!

**Comment by AndyDiags** [comment]
Wow! Thank you for such a quick reply!


> I was able to get it close enough if i trimmed the jumpers off the bluepill 

Yeah agreed, those jumpers need to go, remove and solder a wire.  Such a waste of good space.

One possible option would be to place the SCSI terminal block under the pin connectors, like this so that it would fit under the lip.  This would give it a real slick look.
<img width="497" alt="Screen Shot 2021-11-05 at 11 33 46 PM" src="https://user-images.githubusercontent.com/85074233/140527706-16d93de1-17c1-41be-aa7b-2cc572742738.png">

Taking a quick look at the Macintosh classic, this same would apply, except flip the db-25 connector upside down.

It may not be economical to have many multiple configurations like this available, especially with the added overhead for future enhancements,  but have such customized boards would make it more attractive. 

I did also see some connectors that do provide extra clearance like this one which folks with a Macintosh Classic could use, the lip on it doesn't seem as deep as a Macintosh Plus:  
<img width="631" alt="Screen Shot 2021-11-05 at 11 48 40 PM" src="https://user-images.githubusercontent.com/85074233/140529879-900718cf-946d-4c57-8fbe-c9c9d89d3650.png">
